src/apps/user/templates/profile.html

```diff
@@ -0,0 +1,13 @@
+
+{% block title %}Profile Page{% endblock title %}
+{% block content %}
+    <h1>User Profile Detail</h1>
+
+  <p>Username: {{ user.username }}</p>
```

Comment:
why h1 tag and p tags have a different indentation?

Reply:
Cos we have all this fields in User model but i believe the name of the page should be called profile details, also in the future we can add fields from UserProfile model.

Reply:
Oh sorry my bad, i lost the meaning in translation, indentation fixed :)